integration-test/src/main/scala/com/cloudera/livy/test/framework/BaseIntegrationTestSuite.scala

```diff
@@ -97,15 +97,17 @@ abstract class BaseIntegrationTestSuite extends FunSuite with Matchers with Befo
     }
   }
 
-  before {
+  // We need beforeAll() here because BatchIT's beforeAll() has to be executed after this.
+  // Please create an issue if this breaks test logging for cluster creation.
+  protected override def beforeAll() = {
     cluster = Cluster.get()
     httpClient = new AsyncHttpClient()
     livyClient = new LivyRestClient(httpClient, livyEndpoint)
   }
 
   test("initialize test cluster") {
-    // Empty test case to separate time spent on creating cluster in before() and executing actual
-    // test cases.
+    // Empty test case to separate time spent on creating cluster in beforeAll() and
```

Comment:
This isn't actually needed with `beforeAll`, since that code runs as part of test discovery. So if making that change, this "test" can go away.